build/wasm.js

```diff
@@ -59,50 +80,25 @@ if (hasApk) {
     console.log('Failed to generate build environment information')
     process.exit(-1)
   }
-  writeFileSync(join(WASM_OUT, 'wasm_build_env.txt'), buildInfo)
-}
-
-const writeWasmChunk = EXTERNAL_PATH
-  ? (path, dest) => {
-      const base64 = readFileSync(join(WASM_OUT, path)).toString('base64')
-      writeFileSync(join(WASM_OUT, dest), `
-const { Buffer } = require('node:buffer')
-
-module.exports = Buffer.from('${base64}', 'base64')
-`)
-    }
-  : (path, dest) => {
-      writeFileSync(join(WASM_OUT, dest), `
-const { fs } = require('node:fs')
-
-module.exports = fs.readFileSync(require.resolve('./${basename(path)}'))
-`)
-    }
-
-// Build wasm binary
-execSync(`${WASM_CC} ${WASM_CFLAGS} ${WASM_LDFLAGS} \
- ${join(WASM_SRC, 'src')}/*.c \
- -I${join(WASM_SRC, 'include')} \
- -o ${join(WASM_OUT, 'llhttp.wasm')} \
- ${WASM_LDLIBS}`, { stdio: 'inherit' })
-
-writeWasmChunk('llhttp.wasm', 'llhttp-wasm.js')
-
-// Build wasm simd binary
-execSync(`${WASM_CC} ${WASM_CFLAGS} -msimd128 ${WASM_LDFLAGS} \
- ${join(WASM_SRC, 'src')}/*.c \
- -I${join(WASM_SRC, 'include')} \
- -o ${join(WASM_OUT, 'llhttp_simd.wasm')} \
- ${WASM_LDLIBS}`, { stdio: 'inherit' })
-
-writeWasmChunk('llhttp_simd.wasm', 'llhttp_simd-wasm.js')
-
-if (EXTERNAL_PATH) {
-  writeFileSync(join(ROOT, 'loader.js'), `
-'use strict'
-
-globalThis.__UNDICI_IS_NODE__ = true
-module.exports = require('node:module').createRequire('${EXTERNAL_PATH}/loader.js')('./index-fetch.js')
-delete globalThis.__UNDICI_IS_NODE__
-`)
```

Comment:
^ I meant this.

Reply:
Ok, then it has to be added back + a big fat comment to not remove it because of fedora